src/libs/actions/App.js

```diff
@@ -168,6 +168,24 @@ function reconnectApp() {
     });
 }
 
+/**
+ * Clear downloads
+ */
+function clearDownloads() {
+    const connectionID = Onyx.connect({
+        key: ONYXKEYS.COLLECTION.DOWNLOAD,
+        waitForCollectionCallback: true,
+        callback: (records) => {
+            Onyx.disconnect(connectionID);
+            const downloads = {};
+            _.each(_.keys(records), recordKey => downloads[recordKey] = null);
+            if (!_.isEmpty(downloads)) {
+                Onyx.multiSet(downloads);
+            }
+        },
+    });
+}
+
```

Comment:
I think we should move this to `actions/Download.js` instead

Reply:
Makes sense!